tesler-core/src/main/java/io/tesler/core/service/impl/AccessServiceImpl.java

```diff
@@ -39,9 +40,13 @@
 
 	private final SessionService sessionService;
 
-	public AccessServiceImpl(GroupService groupService, JpaDao jpaDao, SessionService sessionService) {
-		super(groupService, jpaDao, sessionService);
+	private final EffectiveUserAware effectiveUserAware;
+
+	public AccessServiceImpl(GroupService groupService, JpaDao jpaDao, SessionService sessionService,
+			EffectiveUserAware effectiveUserAware) {
+		super(groupService, jpaDao, effectiveUserAware);
 		this.sessionService = sessionService;
+		this.effectiveUserAware = effectiveUserAware;
```

Comment:
Please explain this refactoring in "widget meta loading" feature

Reply:
I took method from EffectiveUserAware interface to separate class to avoid cyclical dependencies in meta-reload process ( class, that had this method before, was called by listeners )